### ISSUE TITLE
[Bug] External playlists limited to first 20 videos

### ISSUE THREAD

**Issue by costinmx** [yuliskov/SmartTubeNext#1188]
Opening most playlists (ie. 'Recommended playlists') in Music section only shows/plays first 20 videos. 
Own (user) playlists shows and plays all the videos.

**Comment by costinmx** [comment]
v13.92, still an issue

**Comment by MTLVMPR** [comment]
This also happens when you add videos to queue. It just stops playback and if you force it to play video 21 then the next video will be the first in the queue.

**Comment by driscollw** [comment]
Yeah, I'm seeing this too on 17.62. Thought maybe the 20 was a default setting I could change. Mostly the music playlist is where this is causing the most issues with playlist being often 50+ videos
